README.md

```diff
@@ -21,7 +21,7 @@ $ helm repo add kedacore https://kedacore.github.io/charts
 $ helm search repo kedacore
 NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
 kedacore/external-scaler-azure-cosmos-db        0.1.0           0.1.0           Event-based autoscaler for Azure Cosmos DB chan...
-kedacore/keda                                   2.12.0          2.12.0          Event-based autoscaler for workloads on Kubernetes
+kedacore/keda                                   2.13.0          2.12.0          Event-based autoscaler for workloads on Kubernetes
```

Comment:
I was told that the app version should not change

Reply:
But this is the chart version

Reply:
Nope it's still https://github.com/kedacore/charts/blob/56f161ca0f50e47eaf852dc344ad15914caeab18/keda/Chart.yaml#L11 We do not need to bump chart version, maintainers take care of that :)

Reply:
Reverted it anyway